### DISCUSSION TITLE
Excluding titles such as 'extended' from collections

### DISCUSSION THREAD

**Discussion by twhiting9275** [meisnate12/Plex-Meta-Manager#281] [Q&A] [answered]
I have a few movies with extended versions (most notably the middle earth stuff), and it seems that both normal and extended versions of the movies are added to collections. I guess there has to be a way to do this, but not sure how.

1: Add a collection with just  the normal movies , based off of a TMDB list
2: Add a collection with just the EXTENDED movies, based off of a TMDB list

The movies themselves have 'extended' in the title, so , not sure if this helps that process or not? Plex sees them as separate movies, since they're split off, and labeled as extended

**Comment by meisnate12** [comment, marked as answer]
i added the `title` collection filter which should help you

**Reply by twhiting9275** [reply]
Thank you :)
